adsingestp/parsers/jats.py

```diff
@@ -555,6 +555,7 @@ def _parse_title_abstract(self):
                 for df in title.find_all("fn"):
                     title_fn_list.append(self._detag(df, self.JATS_TAGSET["abstract"]).strip())
                     df.decompose()
+                art_title = self._detag(title, self.JATS_TAGSET["title"]).strip()
                 if title_group.find("subtitle"):
                     subtitle = title_group.find("subtitle")
                     for dx in subtitle.find_all("ext-link"):
```

Comment:
Are `self.subtitledoi` and `subtitle_xref_list` used anywhere? If not, this logic can be removed

Reply:
subtitledoi can be removed.  Unfortunately there can be xrefs with subtitles, as in `stubdata/input/jats_a+a_subtitle.xml` but there's no room in the subtitle data element in adsingestschema.Document, it's just plain text.  In classic, title footnotes are appended to the abstract (not title), and I'm going to confirm with CSG if that's still behavior we want to continue.  I'd rather update ingestschema to add a structure for subtitle data; if we want it to be merged into the abstract I think that should happen downstream during enrichment, not parsing.

Reply:
This has now been resolved and tested.